cfgov/legacy/views/__init__.py

```diff
@@ -23,11 +22,11 @@ def dbrouter_shortcut(request, content_type_id, object_id):
                 {'ct_id': content_type_id}
             )
 
-        if content_type.app_label in cfgov_apps:
-            obj = content_type.get_object_for_this_type(pk=object_id)
-        else:
+        if content_type.app_label in settings.LEGACY_APPS:
             obj = content_type.model_class().objects.db_manager(
                 'legacy').get(pk=object_id)
+        else:
+            obj = content_type.get_object_for_this_type(pk=object_id)
```

Comment:
Would you mind adding a unit test for this? I'm not quite clear what this is doing. Is the idea that if it's not a legacy app it just uses the default object, but if it is it uses the `legacy` DB? What if this code is hit in a deployment that doesn't have a `legacy` DB (e.g. one where `DATABASE_ROUTING` isn't set)?